render/fx_renderer/fx_pass.c

```diff
@@ -1015,6 +1026,219 @@ bool fx_render_pass_add_optimized_blur(struct fx_gles_render_pass *pass,
 	return !failed;
 }
 
+static void render_drop_shadow_blur_direction(struct fx_gles_render_pass *pass,
+		const struct fx_render_drop_shadow_options *fx_options,
```

Comment:
I'm wondering if instead we just pass the color, base options, and sigma since that's all we use.